app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -194,9 +194,8 @@ object AppRepository {
         if (shifts.isEmpty()) {
             return
         }
-        val timeZoneOffset = shifts.first().timeZoneOffset
         val dayRanges = loadLecturesForAllDays(includeEngelsystemShifts = false)
-                .toDayRanges(timeZoneOffset)
```

Comment:
The members startsAt, endsAt of DayRange are now UTC based. Its still possible to use myDayRange.contains(anyZonedDateTime), since internally the comparison is based on epoch, see org.threeten.bp.chrono.ChronoZonedDateTime#compareTo and DayRangeTest.

Reply:
Thank you. I think I understand it now. I did also not see that the `timeZoneOffset` parameter was unused since 4b2b41e4bdb3547ee74c2070056c59cabd70c1e8. You could reference this commit in the commit message of f232c6ef1ca0cf8cc4a59da96db7205fb504aec3 to ease understanding for the reader.

Reply:
Good point, I just squashed these two commits.